docusaurus/docs/learn/quickstarts/network/help/quickstart-walkthrough.md

```diff
@@ -0,0 +1,314 @@
+---
+title: Quickstart Walkthrough
+id: quickstart-walkthrough
+---
+The following will walk you through the manual process of creating an overlay network with your own PKI. These steps are the same steps that take place behind the scenes when running the OpenZiti Quickstart’s `expressInstall` function.
+
+## General Environment Setup
+
+### Declare Variables
+
+Some global variables used throughout the overlay setup.
+
+- `ZITI_HOME` is the directory where your network files will be stored, the directory will be created if it does not exist already.
+- `ZITI_BIN_DIR` is the directory where the ziti binary will be downloaded and extracted
+- `ZITI_PWD` is the password that will be used when logging into the controller as the default user (admin).
+- `ZITI_HOSTNAME` is used throughout the setup for naming files, network elements, etc.
+
+```
+export ZITI_HOME="${HOME}/.ziti"
+export ZITI_BIN_DIR="${ZITI_HOME}/ziti_bin"
+export ZITI_PWD=admin2
+export ZITI_HOSTNAME=$(hostname -s)
+```
+
+### Create Environment
+
+Create a directory where you will store your network files.
+
+```
+mkdir -p "${ZITI_HOME}"
+```
+
+### Obtain Ziti Binary
+
+The ziti binary is required to setup the network. The exact URL will differ depending on your operating system and architecture. Visit the [releases](https://github.com/openziti/ziti/releases) page to get the appropriate URL.
+
+```
+mkdir -p "${ZITI_BIN_DIR}"
+curl -L <release-url> -o "${ZITI_BIN_DIR}/ziti-bin.gz"
+tar -xf "${ZITI_BIN_DIR}/ziti-bin.gz" --directory "${ZITI_BIN_DIR}"
+chmod +x "${ZITI_BIN_DIR}/ziti"
+```
+
+## Create PKI
+
+You may use an existing PKI if you prefer, however, as part of the `expressInstall` a PKI is generated for you. The following will run through the process of generating your own PKI.
+
+### Setup
+
+Set some initial environment variables for setting up the PKI. The `CA_NAME` values can be any name of your choosing.
```

Comment:
These are all optional. After reading this far, I don't think we should get into the depths of the options the quickstart provides OTHER than what we show people on the host it anywhere page. I think it would be worthwhile to add more doc around the quickstart and all these variables that allow people to customize the quickstart install, but not here

Reply:
What do you mean these are optional? Is this walkthrough just explaining what quickstart does? I thought we were providing the steps to mimic `expressInstall` "manually".

Reply:
i was learning towards an explainer of the quickstart and the "five" or so things people _actually_ change (usually)
```
export EXTERNAL_IP="$(curl -s eth0.me)"       
export ZITI_CTRL_EDGE_IP_OVERRIDE="${EXTERNAL_IP}"
export ZITI_ROUTER_IP_OVERRIDE="${EXTERNAL_IP}"
export ZITI_CTRL_EDGE_ADVERTISED_ADDRESS="${EXTERNAL_DNS:-${EXTERNAL_IP}}"
export ZITI_ROUTER_ADVERTISED_ADDRESS="${EXTERNAL_DNS:-${EXTERNAL_IP}}"
export ZITI_CTRL_ADVERTISED_PORT=8440
export ZITI_CTRL_EDGE_ADVERTISED_PORT=8441
export ZITI_ROUTER_PORT=8442
```

these mostly. we can go on to explain that the express install makes a pki for control plane, data plane and api but not how to customize it. that's more detaill than i wanted in this particular guide. i want them to understand what it does, not how to customize it.

then, having all these type of variables doc'ed in some reference doc telling them how to tweak them all makes sense to me.

that help?

Reply:
Okay, so this is more of an explanation rather than a "this is how you can mimic the quickstart manually"? So, rather than the env vars, should I put hard-coded values for the PKI creation?